naughty-prune

```diff
@@ -53,10 +53,6 @@ def run(unused, verbose=False, **kwargs):
     # The head
     head = execute("git", "rev-parse", "HEAD").strip()
 
-    # Don't replace this issue with the pull request(s)
-    if "issue" in kwargs:
```

Comment:
I mean, really, we should start converting these "tasks" scripts into normal Python...